sycl/test/check_device_code/fpga_datapath_global.cpp

```diff
@@ -13,12 +13,9 @@ const intel::fpga_datapath<int[10]> empty{};
 // CHECK: %[[datapath:.*]] = type { [10 x i32] }
 // CHECK: {{.*}}empty = internal addrspace(1) constant %[[datapath]] zeroinitializer, align 4, !spirv.Decorations ![[empty_md:[0-9]*]]
 
-int main() {
-  queue Q;
+SYCL_EXTERNAL void fpga_datapath_global() {
```

Comment:
JFYI: I think `f` should have been made an argument for the change to be a no-op (it was previously a kernel argument because of the lambda capture), but it likely doesn't matter here.